### PR TITLE
Create excersice menu mobile view

### LINKED ISSUE #83: Create excersice mobile view
## REQUIREMENTS / DESCRIPTION

Create mobile view with corresponding routes
## DESIGN

image

### PR DESCRIPTION
**Pull request by etormasa**
# Description

Create menu with corresponding routes

## Screenshots

image

### CODE REVIEW

**Review by PudinDeveloper — APPROVED** [review]
LGTM

**Review by MauricioNC — APPROVED** [review]

File: app/controllers/selector_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class SelectorController < ApplicationController
+    before_action :authenticate_user!
+    def index
+    end
```
Comment (BlazeDex) [inline app/controllers/selector_controller.rb]: Change to `def index; end`

**Review by Zama15 — APPROVED** [review]
LGTM

File: app/controllers/selector_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  end
+
```
Comment (juperameza) [inline app/controllers/selector_controller.rb]: Add a new line at the end of the file

Comment (juperameza) [reply app/controllers/selector_controller.rb]: aslo what is selector??? what is the purpose of this controller

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by BlazeDex — APPROVED** [review]

**Review by angel23v — APPROVED** [review]
Please don't forget to link the issue for every PR that you do

**Review by juperameza — APPROVED** [review]
Lgbt

**Pull request merged**